### PR TITLE
[PR] Force the front page to use a Crimson Spine

### PR DESCRIPTION
**Pull request by philcable**

### CODE REVIEW

**Comment by philcable** [comment]
@jeremyfelt #reviewmerge?

File: functions.php [hunk 1]
```diff
@@ -41,5 +42,32 @@ public function enqueue_scripts() {
...
+	 * @param string $option The current Spine option being filtered.
+	 *
+	 * @return string The modified Spine color.
+	 */
+	public function filter_spine_color( $value, $option ) {
+		if ( 'spine_color' !== $option ) {
+			return $value;
+		}
+
+		$site = get_blog_details();
```
Comment (jeremyfelt) [inline functions.php]: Can you do `get_site()` instead? I want to deprecate `get_blog_details()` in WP core one day. :)

**Review by jeremyfelt — CHANGES_REQUESTED** [review]
One small selfish change :)

File: functions.php (hunk 1 above)
Comment (philcable) [reply functions.php]: Sure thing!

**Review by jeremyfelt — APPROVED** [review]

**Pull request merged**